pkgs/tools/security/bitwarden/default.nix

```diff
@@ -128,6 +89,41 @@ buildNpmPackage' {
     popd
   '';
 
+  doCheck = true;
+
+  nativeCheckInputs = [
+    dbus
+    (gnome.gnome-keyring.override { useWrappedDaemon = false; })
+  ];
+
+  checkFlags = [
+    "--skip=password::password::tests::test"
+  ];
+
+  checkPhase = ''
+    runHook preCheck
+
+    pushd ${cargoRoot}
+    export HOME=$(mktemp -d)
+    export -f cargoCheckHook runHook _eval _callImplicitHook
+    export cargoCheckType=release
+    dbus-run-session \
+      --config-file=${dbus}/share/dbus-1/session.conf \
+      -- bash -e -c cargoCheckHook
+    popd
+
+    runHook postCheck
+  '';
+
+  desktopItem = makeDesktopItem {
+    name = "bitwarden";
+    exec = "bitwarden %U";
+    inherit icon;
+    comment = meta.description;
```

Comment:
we shouldn't do this as changing meta should not trigger a rebuild

Reply:
It was the same before. If you change `meta.description` you can use the string verbatim here instead.

Reply:
Sure but that doesn't fix the problem 😅 If I am opening now a PR than either of us hits a merge conflict and I also want to avoid that.

Reply:
> If I am opening now a PR than either of us hits a merge conflict and I also want to avoid that.

That's ridiculous. No other PR targetting bitwarden is open right now.

Anyway, I'll move `desktopItem` and `description` out of `buildNpmPackage`.